airflow/serialization/serialized_objects.py

```diff
@@ -346,6 +348,9 @@ def _deserialize(cls, encoded_var: Any) -> Any:
             return {cls._deserialize(v) for v in var}
         elif type_ == DAT.TUPLE:
             return tuple(cls._deserialize(v) for v in var)
+        elif type_ == DAT.PARAM:
+            param_class = import_string(var['_type'])
+            return param_class(**var)
```

Comment:
Isn't this doing to do the effect of `Param(_type="param", ...)` -- i.e. pass invalid kwargs to Param constructor?

Reply:
yeah, it would be. though it's harmless but will fix this.